packages/marketplace/ui/src/lib/hooks/use-order.tsx

```diff
@@ -5,18 +5,33 @@ import {
 } from "@/shared/constants/marketplace";
 import {
   OfferMetadata,
+  Order,
   OrderDeploymentMetadata,
   OrderSetupMetadata,
 } from "@/types";
 import { useReadContract } from "wagmi";
 
+
+const safeParseJSON = (value: string | null | undefined) => {
+  try {
+    return value ? JSON.parse(value) : {};
+  } catch {
+    return {};
+  }
+};
```

Comment:
We should move to utils but still don't know why we want a safe parse, if we don't have metadata we cannot render anything so order is corrupted